modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/HealthCheckRound.scala

```diff
@@ -0,0 +1,5 @@
+package org.tessellation.sdk.domain.healthcheck
+
+class HealthCheckRound[F[_]] {
```

Comment:
Not sure it can be a `class` if there will be different kinds of healthchecks for different `healthcheckKeys` 🤔